sql/core/benchmarks/CollationBenchmark-jdk21-results.txt

```diff
@@ -1,54 +1,88 @@
+UTF8_LCASE                                        33605          33643          54          0.0      336053.8       1.9X
+UNICODE                                          916945         917042         139          0.0     9169445.3      52.3X
+UNICODE_CI                                       992431         992565         189          0.0     9924305.9      56.7X
 
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.5.0-1025-azure
-AMD EPYC 7763 64-Core Processor
+OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-aws
+Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
 collation unit benchmarks - endsWith:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                       10551          10556           7          0.0      105511.7       1.0X
-UTF8_LCASE                                        20294          20300           9          0.0      202943.7       1.9X
-UNICODE                                          384070         384554         684          0.0     3840704.6      36.4X
-UNICODE_CI                                       441935         442184         352          0.0     4419351.4      41.9X
+UTF8_BINARY                                       17482          17485           5          0.0      174819.0       1.0X
+UTF8_LCASE                                        33890          33925          49          0.0      338901.6       1.9X
+UNICODE                                          929570         930339        1087          0.0     9295702.4      53.2X
+UNICODE_CI                                      1002779        1003427         916          0.0    10027792.9      57.4X
+
+OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-aws
+Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
+collation unit benchmarks - initCap using impl execICU:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
+--------------------------------------------------------------------------------------------------------------------------------------
+UNICODE                                                            10             11           1          9.7         102.6       1.0X
+UNICODE_CI                                                         12             12           1          8.4         118.8       1.2X
+
+OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-aws
+Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
+collation unit benchmarks - initCap using impl execBinaryICU:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
+--------------------------------------------------------------------------------------------------------------------------------------------
+UTF8_BINARY                                                               2              2           0         64.6          15.5       1.0X
```

Comment:
Could you increase the number of iterations to have non-zero `StdDev`, and make the benchmark more reliable.

Reply:
Sorry. Fixed. 

I re-evaluated just initCap related benchmarks.

Reply:
As we can see, this benchmark is sensitive to CPU clock speed. During my latest measurements on an `Intel(R) Xeon(R) Platinum 8252C CPU @ 3.80GHz` (AWS m5zn.xlarge), the stdev for some measurements - along with others - dropped to zero or one.

I suggest adding more decimal places to the results in a separate PR.